tomte/tools/check_readme.py

```diff
@@ -39,10 +39,14 @@ def main(service_package: str) -> None:
         if wanted_hash is None or wanted_hash not in readme:
             # we were not able to find the wanted hash
             not_found = (
-                "The ElCollectooorr service package hash was not found on the readme."
+                f"The '{service_package}' service package hash was not found on the readme."
             )
             sys.exit(not_found)
 
 
 if __name__ == "__main__":
```

Comment:
Why is this needed at all?

Reply:
Do you mean the parameter? It is unnecessary since the service package will always match the input.
Removed on 5ed321f99f218b3c1ddfe59e97ca034281ad4530.

Reply:
I mean the ability to run it as a script. This is called by CLI entrypoint so we don't need the `if __name__ == "__main__":`

Reply:
Removed on 28ab70eb8e71439b49e7933db733b52550a03511.